tests/posix/headers/src/net_if_h.c

```diff
@@ -19,15 +19,15 @@
  */
```

Comment:
About the commit `tests: posix: headers: uncomment net/if.h tests`,
could you add a slightly more informative commit text, the current one states the obvious, but what should be mentioned here why we need/want to uncomment the test.
Same comment to previous commit about the netdb.h

Reply:
Fixed!